test/main.js

```diff
@@ -23,14 +21,10 @@ function init(callback) {
 	require('./../app');
 
 	// Run the battle engine in the main process to keep our sanity
-	let BattleEngine = global.BattleEngine = require('./../battle-engine');
 	for (let listener of process.listeners('message')) {
 		process.removeListener('message', listener);
 	}
```

Comment:
Well, this is currently dead code, so I can see how it can be considered out of scope,
but since it existed because of the line above, it can be also argued otherwise.

Anyway, the comment above should go
> // Run the battle engine in the main process to keep our sanity